app/workers/retirement_worker.rb

```diff
@@ -15,8 +15,15 @@ def perform(count_id)
 
   def deactivate_workflow!(workflow)
     if workflow.finished?
-      workflow.active = false
-      workflow.save!
+      ignore_optimistic_lock { workflow.update_attribute(:active, false) }
     end
   end
+
+  private
+
+  def ignore_optimistic_lock
+    ActiveRecord::Base.lock_optimistically = false
```

Comment:
Yeah I think if you use the class method `::update` it skips looking so you could do 

```ruby
Workflow.update(workflow.id, {activate: false})
```
but I can't find any docs to back up my memory.